src/CachePlugin.php

```diff
@@ -45,6 +45,7 @@
      *     @var int $cache_lifetime (seconds) To support serving a previous stale response when the server answers 304
      *              we have to store the cache for a longer time than the server originally says it is valid for.
      *              We store a cache item for $cache_lifetime + max age of the response.
+     *     @var array $methods case sensitive list of request methods which can be cached.
```

Comment:
Cache plugin also [converts the request method to uppercase](https://github.com/php-http/cache-plugin/blob/master/src/CachePlugin.php#L65). While this is not technically correct it is probably ok for 99% of the use cases.